src/test/java/org/mule/extension/validation/BasicValidationTestCase.java

```diff
@@ -158,7 +157,6 @@ public void notEmpty() throws Exception {
     assertInvalid(flowRunner(flow).withPayload(new Object[] {}), messages.arrayIsEmpty());
     assertInvalid(flowRunner(flow).withPayload(new int[] {}), messages.arrayIsEmpty());
     assertInvalid(flowRunner(flow).withPayload(new HashMap<String, String>()), messages.mapIsEmpty());
-    assertInvalid(flowRunner(flow).withPayload(BlankLiteral.INSTANCE), messages.valueIsBlankLiteral());
```

Comment:
can't the same behabior be testes using `""` instead of the MEL constant?

Reply:
no, valueIsBlankLiteral is checking against BlankLiteral class. I removed that code that is not usable anymore